introduction.md

```diff
@@ -3,3 +3,5 @@ My favorite Code School path is Javascript.
 *Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?